perfkitbenchmarker/linux_benchmarks/mysql_service_benchmark.py

```diff
@@ -840,9 +863,26 @@ def Prepare(benchmark_spec):
   # Setup common test tools required on the client VM
   vms[0].Install('sysbench05plus')
 
+
+  vms[0].mysql_svc_oltp_tables_count = FLAGS.mysql_svc_oltp_tables_count
```

Comment:
+1 

I think these do belong to the benchmark_spec object, table count and table size are exactly "benchmark spec". ;)

(Sam: it's my bad on suggesting putting these to the VM object, benchmark_spec is much better.